query/QueryTranslator.java

```diff
@@ -1,16 +1,11 @@
-package query;
-
-import datasets.Dataset;
-
 import java.sql.Timestamp;
-import java.util.List;
 
 public abstract class QueryTranslator {
     public abstract String translate(QueryType type);
 
     public abstract String translateCreateTable(Dataset dataset);
 
-    public abstract String translateInsertInto(Dataset dataset, List<String> values);
+    public abstract String translateInsertInto(Dataset dataset, String[][] values);
```

Comment:
is there a reason for `String[][]` instead of `List<List<String>>`? If this List interface is easier to use

Reply:
we have the size of the array/list given before, in this case arrays have a better performance I think